### DISCUSSION TITLE
How to extract Bayes factor from output? (mismatch)

### DISCUSSION THREAD

**Discussion by jtruhs** [easystats/bayestestR#441] [Q&A] [answered]
Maybe a simple question but here's a simple reproducible example (it happens with any example for me though):

```
tab <- as.table(cbind(Office=c(18,31),Phone=c(5,7)))
ctb <- contingencyTableBF(tab,sampleType = "poisson")
ctb
Bayes factor analysis
--------------
[1] Non-indep. (a=1) : 0.4968786 ±0%

Against denominator:
  Null, independence, a = 1 
---
Bayes factor type: BFcontingencyTable, poisson

ctb@bayesFactor$bf
[1] -0.6994095
```

I'm trying to just extract the Bayes factor of 0.4969 programmatically. When I call from the class `@bayesFactor$bf` it gives me a different number, and I'm not even sure it makes sense that Bayes factors can be negative?

Am I doing something wrong, and if so, what is the proper way to gather that number from the results? Thanks for your help

**Comment by bwiernik** [comment, marked as answer]
You're extracting the log Bayes factor. Internally it's stored that way for computational stability when the BF gets very large/small. Covert to a BF with `exp()`.

**Reply by jtruhs** [reply]
Thank you very much. That makes so much more sense.
